src/redux/Books/Books.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
- [ ] great job so far. Please ensure that createAsyncThunk is used in the project for creating thunk action creators for the different app functions.